client/dashboard/index.js

```diff
@@ -15,25 +15,37 @@ import Leaderboards from './leaderboards';
 import { ReportFilters } from '@woocommerce/components';
 import StorePerformance from './store-performance';
 import TaskList from './task-list';
+import ProfileWizard from './profile-wizard';
 
 export default class Dashboard extends Component {
 	render() {
 		const { query, path } = this.props;
+
+		// @todo This should be replaced by a check to see if the wizard was completed or skipped (#1884, #1883).
+		const profileWizardComplete = true;
 		// @todo This should be replaced by a check of tasks from the REST API response from #1897.
-		const requiredTasksComplete = false;
+		const requiredTasksComplete = true;
+
+		let dashboardOutput;
```

Comment:
Optional refactor: we could put this into a `renderDashboardOutput()` function and return the output block to avoid using `let`.